pacstall

```diff
@@ -953,11 +952,6 @@ Helpful links:
                 exit 1
             }
 
-            if curl --output /dev/null --silent --head --fail https://raw.githubusercontent.com/"$USERNAME"/pacstall/"$BRANCH"/pyproject.toml || curl --output /dev/null --silent --head --fail https://raw.githubusercontent.com/"$USERNAME"/pacstall/"$BRANCH"/Cargo.toml; then
```

Comment:
This section is still required. Even though no upstream branch exists for these, if someone is working on pacstall-py or pacstall-rs on a repo called simply pacstall, this prevents an upgrade to that repo.

Reply:
A rewrite is not even happening currently. If we're gonna cover entirely hypothetical situations, might as well add this check for every programming language. Makes no sense and claiming such a check is "required" is absurd

Reply:
https://github.com/pacstall/pacstall/pull/460 it's been there for years. there's no reason to be removing it now. there equally makes no sense for you not to just leave it alone.

Reply:
It's at least one redundant curl query, which makes for worse performance. That's why I removed it

Reply:
It's not redundant. It's possible in the future we might have a rewrite still, I say we should not remove this.